### DISCUSSION TITLE
How to fix permission issues in Windows caused by Latest version of Quarto-cli

### DISCUSSION THREAD

**Discussion by Rahuketu86** [quarto-dev/quarto-cli#7165] [Q&A]
### Description

![image](https://github.com/quarto-dev/quarto-cli/assets/786803/a9235cde-1269-469d-a9dd-4fa2446421a7)

I have now checked this on multiple machines and with windows 10 and windows 11 operating system. 

Latest upgrade to version 1.3.450 is giving me above error. Please help me fix this issue

**Comment by Rahuketu86** [comment]
I have already changes permissions of quarto folder to read, write and execute

**Comment by mcanouil** [comment]
How did you install Quarto as there are several possible ways which does not do the exact same thing?

Also, what if you try in another local directory (not on cloud/network drives of any kind)?

What's the output of `quarto check`?

See the troubleshooting guide at <https://quarto.org/docs/troubleshooting/index.html>.

**Reply by Rahuketu86** [reply]
How did you install Quarto as there are several possible ways which does not do the exact same thing?
> I have tried both ways for windows - 1.system wide install with Admin permission , 2. Local install for individual user . Both result in same error 

Also, what if you try in another local directory (not on cloud/network drives of any kind)?
>  Current install is in default user directory ( above image)
C:/Users/Rahuk<User>/AppData/Local/Bin/Quarto

What's the output of quarto check?

Quarto check runs without any issue

<img width="266" alt="image" src="https://github.com/quarto-dev/quarto-cli/assets/786803/5d92979d-6c20-40de-bb14-b38e8c1311de">



I have tried this now with 
1. reinstalling anaconda to latest version
2. Using mamba miniforge 
3. 2 different laptops 

Same error everytime( something has changed since Quarto version 1.2x which ran without any issues on windows

**Reply by mcanouil** [reply]
Thanks for the additional information!

What I meant about the install process was did you use the msi asset, the archive asset, Scoop package manager or Chocolatey, etc.
